client/src/includes/messages.test.js

```diff
@@ -0,0 +1,99 @@
+import { initButtonSelects } from './initButtonSelects';
+
+// save our DOM elements to a variable
+const testElements = `
+<div class = "button-select">
+  <input type="hidden" value/>
```

Comment:
maybe start your DOM with no `value` or `value=""` - it will make it clearer what you are doing. Remember that the browser puts this empty attribute in this way but this can cause confusion.